### PR TITLE
Added npm install requirement to the README

### PR DESCRIPTION
**Pull request by rbillings**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -172,6 +172,12 @@ Then fill in the generated file with logic, fixtures, etc. You can then apply th
 
 Running the Development Server
 ------------------------------
+You will need to have the Node.js package manager npm installed. You can install it by running:
```
Comment (bobsilverberg) [inline README.md]: This isn't a totally accurate description. We're not installing the Node.js package manager (npm), we are in fact using npm to install other dependencies that are required for One and Done.

**Comment by davehunt** [comment]
Lots of commits here - possibly needs a rebase @rbillings?

**Comment by rbillings** [comment]
Rebased and ready for a review, @mozilla/web-qa-sorcerers ?

**Comment by stephendonner** [comment]
Still that big user.json blog, @rbillings - https://github.com/mozilla/oneanddone/pull/341/files#diff-f3e194e100c9d799628b115d21a2e169

**Comment by rbillings** [comment]
@stephendonner - thanks for looking at this. I just added comments to the README regarding the npm install, I'm unclear what you are referencing for the user.json file?

**Comment by rbillings** [comment]
Thanks for noticing that extra file that snuck in!

**Pull request merged**